packages/backend/server/src/core/auth/controller.ts

```diff
@@ -156,6 +150,8 @@ export class AuthController {
 
     const valid = await this.token.verifyToken(TokenType.SignIn, token, {
       credential: email,
+      // don't revoke the token to avoid email client prefetching touch link
+      keep: true,
```

Comment:
then when will we delete the token?

Reply:
will deleted after click `open affine`/ `submit questionnaire` in onboarding page
frontend change will commit later

Reply:
that makes no sense. the token should be expired once the user open the link.

I forget the mailer prefetching stuff. I will make a new page with and sending a POST request to login to replace this api

Reply:
most email clients provide link preview feature:
![image](https://github.com/toeverything/AFFiNE/assets/25152247/a7665655-d9b6-4f43-99db-73db0c2a5fcc)

and the browser's preloading function may also preload it:
<img width="862" alt="image" src="https://github.com/toeverything/AFFiNE/assets/25152247/9496fe3b-ed96-4a1a-807b-eff11f78acaf">

previously, some users report that the email login link will be invalid randomly, and this has also occurred in our internal tests

Reply:
cool, I will do the workaround, you can revoke it here